Recast/HomeViewController.swift

```diff
@@ -0,0 +1,274 @@
+//
+//  HomeViewController.swift
+//  Recast
+//
+//  Created by Jaewon Sim on 9/19/18.
+//  Copyright © 2018 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+struct DummyPodcastSeries {
+    let image: UIImage
+    let title: String
+    let date: String
+    let duration: Int
+    let timeLeft: Int
+    let isNew: Bool
+
+    init(image: UIImage, title: String, date: String, duration: Int, timeLeft: Int, isNew: Bool) {
+        self.image = image
+        self.title = title
+        self.date = date
+        self.duration = duration
+        self.timeLeft = timeLeft
+        self.isNew = isNew
+    }
+
+    init(image: UIImage, title: String, date: String, duration: Int, timeLeft: Int) {
+        self.image = image
+        self.title = title
+        self.date = date
+        self.duration = duration
+        self.timeLeft = timeLeft
+        self.isNew = false
+    }
+}
+
+enum HomeSectionType: String {
+    case continueListening = "Continue Listening"
+    case yourFavorites = "Your Favorites"
+    case browseTopics = "Browse Topics"
+}
+
+class HomeViewController: UIViewController {
+
+    //dummy structs for testing
+    let dummyContinue1 = DummyPodcastSeries(
+            image: #imageLiteral(resourceName: "series_img_1"),
+            title: "110: Asking All the Questions",
+            date: "September 13, 2018",
+            duration: 58,
+            timeLeft: 10,
+            isNew: true)
+
+    let dummyContinue2 = DummyPodcastSeries(
+        image: #imageLiteral(resourceName: "series_img_2"),
+        title: "Dummy Podcast 2",
+        date: "November 1st, 2018",
+        duration: 10,
+        timeLeft: 29)
+
+    var homeTableView: UITableView!
+
+    /// Maps section/row index to the appropriate header type
+    let headerTypes: [Int: HomeSectionType] = [
```

Comment:
Why is this and `collectionViewTypes` a dictionary? If you're using ints as keys its best to just use an array. Also why can't you use 0, 1, 2 for the collectionView tags?